cfg/gke-1.2.0/policies.yaml

```diff
@@ -20,29 +20,22 @@ groups:
         scored: false
 
       - id: 4.1.2
-        text: "Minimize access to secrets (Manual)"
+        text: "Minimize access to secrets (Automated)"
         type: "manual"
         remediation: |
           Where possible, remove get, list and watch access to secret objects in the cluster.
         scored: false
 
       - id: 4.1.3
-        text: "Minimize wildcard use in Roles and ClusterRoles (Manual)"
+        text: "Minimize wildcard use in Roles and ClusterRoles (Automated)"
```

Comment:
Why changing to Automated in the text when the type is still manual?

Reply:
this is fixed